.github/workflows/pr.yaml

```diff
@@ -97,6 +97,21 @@ jobs:
       - name: Run deployment variable definition validation test
         run: python3 cloud/shared/validate_variable_definitions_test.py
 
+  validate_terraform:
+    runs-on: ubuntu-latest
+    steps:
+      - name: check out pr branch
+        uses: actions/checkout@v3
+      - name: generate bastion key
+        run:  ssh-keygen -q -t rsa -b 4096 -N '' -f "${HOME}/.ssh/bastion"
+      - name: setup terraform
+        uses: hashicorp/setup-terraform@v1
+      # as we add more templates, we will add checks to validate them. for now, we only have one
+      - name: terraform init azure_saml_ses 
+        run: terraform -chdir=cloud/azure/templates/azure_saml_ses init -backend=false 
```

Comment:
ooo does this work like this with backend=false

Reply:
lol i should do this instead of the overrides

Reply:
https://www.terraform.io/cli/commands/validate
"Validation requires an initialized working directory with any referenced plugins and modules installed. To initialize a working directory for validation without accessing any configured remote backend, use:
```
terraform init -backend=false
```
To verify configuration in the context of a particular run (a particular target workspace, input variable values, etc), use the terraform plan command instead, which includes an implied validation check."